src/dasmon_app/dasmon_listener/settings.py

```diff
@@ -48,7 +48,7 @@
 
 INSTALLATION_DIR = "/var/www/workflow/app"
 
-PURGE_TIMEOUT = 7
+PURGE_TIMEOUT = 0.5
```

Comment:
1. Please add units in a comment
2. Why did you shrink it from 7 to 0.5?

Reply:
This is the time that the PVs are purged, production is currently 0.5 days. I don't think we need 10080 minutes of PV plots.